doc/Requirements Specification/Parts/Environment.lyx

```diff
@@ -214,6 +214,30 @@ gls{measurement software}
 .
 \end_layout
 
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/E40/"
+
+\end_inset
+
+ The user should not run other programms while Beagle is running.
+ By reason that it distorts the measurement.
```

Comment:
Maybe following is better:
"The user should not run other programs while Beagle is running as this disturbs the measurement.